pkg/e2e/state/prometheus.go

```diff
@@ -26,6 +31,27 @@ var _ = ginkgo.Describe(clusterStateInformingName, func() {
 		// common for a running prometheus instance
 		cmd := promCollectCmd + " > " + runner.DefaultRunner.OutputDir + "/prometheus.tar.gz; err=$? ; if (( $err != 0 )) ; then exit $err ; fi"
 
+		poderr := wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+			podCount := 0
+			var podErr error
+			list, _ := verify.FilterPods("openshift-monitoring", "app=prometheus", h)
+			names, podNum := verify.GetPodNames("prometheus-operator", list, h)
+			if podNum > 0 {
+				for _, value := range names {
+					if (value == "prometheus-k8s-0") || (value == "prometheus-k8s-1") {
+						podCount += 1
+					}
+				}
+				if podCount == 2 {
```

Comment:
```suggestion
				if podCount >= 2 {
```
What if the default number of pods increases in the future?

Reply:
That could be a possibility. I've implemented the change.